### DISCUSSION TITLE
三方隐私求交是否支持各方求交字段不对齐的情况

### DISCUSSION THREAD

**Discussion by guizgis** [secretflow/secretflow#864] [Q&A]
三方隐私求交，A只有字段uid1，B只有字段uid2，C有字段uid1和uid2的对应关系，能否实现A、B、C三方同时参与的求交

**Comment by wangyixianggogo** [comment]
哈喽，参与方用来求交的id（字段），应该是一致的，具体的内容可以不同，所以你提的情况是不支持的，顺便问下这个场景是有什么具体的需求吗？
